README.md

```diff
@@ -54,91 +65,110 @@ Okay, now you know why to document let's define some rules that everyone has to
 NOTE: The member can either directly push his edits in the central repo on [Github] and the above same procedure is to be followed before the coordinator pulls it in the main repo.
 Or the students can submit the content as **.txt** file and it will be uploaded in the main repo
 
+### Role of Team heads
 
-##### Role of Team heads
 1. Make sure that no two team members have written the same content
 2. If it's a collaborated work; the contribution of each team member with equal effort has to be acknowledged
 3. The content provided by the members does not resemble any previous work [Check central repo].
 
-##### Role of the coordinator
+### Role of the coordinator
+
 1. Collect all the docs from each team head after rectification
 2. Pull requests once the team head gives a green flag
 3. Upload .txt files in the main repo and add the link in front of the respected topics
 4. Handle any discrepancies from the team members
 5. Handle queries from any person trying to know about the content
 6. Handle content from non-club members (Club alumni/experts)
 
+## Mechanical
+
+### Fabrication Engineer
+
+1. Location of acquisition of materials - Each material, exact location, cost with date.
 
-## Mechanical :gear: 
-##### Fabrication Engineer
-1. Location of acquisition of materials - Each material, exact location, cost with date]
-2. 
 ---
-## Electrical/Electronics  
 
-##### Battery Management Engineer
+## Electrical/Electronics
+
+### Battery Management Engineer
 
 The following document has been modified on 18th October 2018
 [Doc given by Abbas bhai](https://github.com/auvzhcet/Documentation/blob/master/BMS%20for%20lithium%20polymer%20battery%20pack.docx)
 
-##### PCB designer
-##### Acoustic Engineer
+### PCB designer
+
+### Acoustic Engineer
 
 [Comparative Study of hydrophones used by other teams](https://github.com/auvzhcet/AUV2k19/blob/master/hydrophones_study.md)
 
 [Our implemetation](https://github.com/auvzhcet/Documentation/blob/master/Acoustics.md)
 
 ---
-## Computer :computer:
```

Comment:
We can use emojis for readme.
Emojis were removed for index.md only as they are not rendered by readthedocs. README is meant for github only.